.expeditor/verify.pipeline.yml

```diff
@@ -190,4 +190,23 @@ steps:
         docker:
           privileged: true
           environment:
-            - HAB_ORIGIN=ci
\ No newline at end of file
+            - HAB_ORIGIN=ci
```

Comment:
Is this needed? It only affects the builder-graph build which should be unrelated to all of this.

Reply:
This is not exactly a change introduced by this commit, this is shown in diff because of the added newline above it. :-)